diff -up qtlocation-opensource-src-5.3.1/src/plugins/position/geoclue/qgeopositioninfosource_geocluemaster.cpp.G_VALUE_INIT qtlocation-opensource-src-5.3.1/src/plugins/position/geoclue/qgeopositioninfosource_geocluemaster.cpp
--- qtlocation-opensource-src-5.3.1/src/plugins/position/geoclue/qgeopositioninfosource_geocluemaster.cpp.G_VALUE_INIT	2014-06-19 05:08:32.000000000 -0500
+++ qtlocation-opensource-src-5.3.1/src/plugins/position/geoclue/qgeopositioninfosource_geocluemaster.cpp	2014-07-24 09:48:59.800638065 -0500
@@ -61,6 +61,10 @@ QT_BEGIN_NAMESPACE
 #define MINIMUM_UPDATE_INTERVAL 1000
 #define UPDATE_TIMEOUT_COLD_START 120000
 
+#ifndef G_VALUE_INIT
+#define G_VALUE_INIT  { 0 }
+#endif
+
 namespace
 {
 
